analysis/cohort_joiner.py

```diff
@@ -16,11 +16,24 @@ def read_dataframe(path):
 
 
 def write_dataframe(dataframe, path):
+    # Aside from argument names and types, we use the same code here as we do in
+    # `cohortextractor.pandas_utils.dataframe_to_file`.
```

Comment:
Smells like an opportunity for a `dataframe_utils` library!

Reply:
@rebkwok reminded me that cohort-extractor is a dependency of python-docker (`opensafely-cohort-extractor`), so I think I'll replace the copy-paste code with a call to `cohortextractor.pandas_utils.dataframe_to_file` in a future PR. (There's still a potential versioning issue, but hopefully the code is stable.)

Reply:
Created #10 as a reminder.